app/models/parser/twitter_search_item.rb

```diff
@@ -0,0 +1,36 @@
+module Parser
+  class TwitterSearchItem < Base
+    include ProviderTwitter
+
+    TWITTER_SEARCH_ITEM_URL = /https:\/\/twitter\.com\/search[^\s"]*/
+
+    class << self
+      def type
+        'twitter_search_item'.freeze
+      end
+
+      def patterns
+        [TWITTER_SEARCH_ITEM_URL]
+      end
+    end
+
+    private
+
+    # Main function for class
+    def parse_data_for_parser(_doc, _original_url, _jsonld_array)
+      handle_exceptions(StandardError) do
+        params = URI.parse(url).query.split('&').map { |pair| pair.split('=') }.to_h
```

Comment:
Request: Testing locally, I noticed that in some cases the URLs we receive contain escaped characters, so I suggest we unescape when parsing the URL here.

![Captura de tela de 2023-12-05 23-35-13](https://github.com/meedan/pender/assets/117518/c3e13b04-bd57-45c0-8478-4437cb893585)